src/Extensions/ImagesExtension.php

```diff
@@ -1,10 +1,66 @@
 <?php
 
-namespace Xefi\Faker\Files\Extensions;
+namespace Xefi\Faker\Images\Extensions;
 
-class ImagesExtension
+use Intervention\Image\Image;
+use Random\Randomizer;
+use Xefi\Faker\Extensions\Extension;
+use Xefi\Faker\Images\Providers\ImageManagerProvider;
+
+class ImagesExtension extends Extension
 {
-    public function image() {
+    private ImageManagerProvider $imageManagerProvider;
+
+    public function __construct(Randomizer $randomizer){
+        $this->imageManagerProvider = new ImageManagerProvider();
+
+        parent::__construct($randomizer);
+    }
+
+    /**
+     * Calculate the best text size from width and height
+     *
+     * @param int $width
+     * @param int $height
+     *
+     * @return int
+     */
+    private function calculateTextSize(int $width, int $height): int
+    {
+        // If the width is 10 times higher than the height, we use width to calculate the text size
+        // Else we use the height
+        if ($width / 10 < $height) {
+            if ($width > 100)
+                $size = $width / 10;
+            else
+                $size = $width / 5;
+        } else {
+            $size = $height / 5;
+        }
```

Comment:
This feels hard to read, also you should always use brackets

$size = max($width, $height) / 5

Why do you divide by 10 in a single treatment ?

Reply:
Reviewed the calcul with you to make it better